notebooks/407-person-tracking-webcam/407-person-tracking-paddle.ipynb

```diff
@@ -0,0 +1,915 @@
+    "    \"\"\"\n",
+    "\n",
+    "    def __init__(self, infer_config):\n",
+    "        # parsing Yaml config for Preprocess\n",
+    "        with open(infer_config) as f:\n",
+    "            yml_conf = yaml.safe_load(f)\n",
+    "        self.check_model(yml_conf)\n",
+    "        self.arch = yml_conf[\"arch\"]\n",
+    "        self.preprocess_infos = yml_conf[\"Preprocess\"]\n",
+    "        self.min_subgraph_size = yml_conf[\"min_subgraph_size\"]\n",
+    "        self.label_list = yml_conf[\"label_list\"]\n",
+    "        self.use_dynamic_shape = yml_conf[\"use_dynamic_shape\"]\n",
+    "        self.draw_threshold = yml_conf.get(\"draw_threshold\", 0.5)\n",
+    "        self.mask = yml_conf.get(\"mask\", False)\n",
+    "        self.tracker = yml_conf.get(\"tracker\", None)\n",
+    "        self.nms = yml_conf.get(\"NMS\", None)\n",
+    "        self.fpn_stride = yml_conf.get(\"fpn_stride\", None)\n",
+    "        if self.arch == \"RCNN\" and yml_conf.get(\"export_onnx\", False):\n",
+    "            print(\n",
+    "                \"The RCNN export model is used for ONNX and it only supports batch_size = 1\"\n",
+    "            )\n",
+    "        self.print_config()\n",
+    "\n",
+    "    def check_model(self, yml_conf):\n",
+    "        \"\"\"\n",
+    "        Raises:\n",
+    "            ValueError: loaded model not in supported model type\n",
+    "        \"\"\"\n",
+    "        for support_model in SUPPORT_MODELS:\n",
+    "            if support_model in yml_conf[\"arch\"]:\n",
+    "                return True\n",
+    "        raise ValueError(\n",
+    "            \"Unsupported arch: {}, expect {}\".format(yml_conf[\"arch\"], SUPPORT_MODELS)\n",
+    "        )\n",
+    "\n",
+    "    def print_config(self):\n",
+    "        print(\"-----------  Model Configuration -----------\")\n",
+    "        print(\"%s: %s\" % (\"Model Arch\", self.arch))\n",
+    "        print(\"%s: \" % (\"Transform Order\"))\n",
+    "        for op_info in self.preprocess_infos:\n",
+    "            print(\"--%s: %s\" % (\"transform op\", op_info[\"type\"]))\n",
+    "        print(\"--------------------------------------------\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "42dd8dae-e611-49ba-916b-13b0405679a0",
+   "metadata": {},
+   "source": [
+    "## Initialize Openvino predictor for PP-YOLOE and OC-Sort\n",
+    "1. initialize the runtime for inference. Then, read the network architecture and model weights from the `.pdmodel` and `.pdiparams` files to load to CPU.\n",
+    "2. initialize the OC-Sort Tracker with default configs"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a37f57a2-788c-430c-b746-d0541f125c43",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class label(object):\n",
+    "    def __init__(self):\n",
+    "        self.labels = [\"pedestrian\"]\n",
+    "\n",
+    "\n",
+    "class OpenvineDetector(object):\n",
```

Comment:
openvino is only for detection inference, and we have oc-sort algorithm for tracking, right ? so my suggestion is to split the detection and track part separately. This class shall be only for openvino detection.